### PR TITLE
docs: 更新截图/润色文档

### PR DESCRIPTION
**Pull request by lcx-seima**
### 👀 PR includes

✨ Feature

- [ ] New feature

🎨 Enhance

- [ ] Code style optimization
- [ ] Refactoring
- [ ] Change the UI
- [ ] Improve the performance
- [ ] Type optimization

🐛 Bugfix

- [ ] Solve the issue and close #0

🔧 Chore

- [ ] Test case
- [x] Docs / demos update
- [ ] CI / workflow
- [ ] Release version
- [ ] Other ()

### 📝 Description
更新“总计”中错误的指示图
润色部分文档

### 🖼️ Screenshot

| Before | After |
| ------ | ----- |
| ❌      | ✅     |

### 🔍 Self Check before Merge

- [x] Add or update relevant Docs.
- [x] Add or update relevant Demos.
- [x] Add or update relevant TypeScript definitions.

### CODE REVIEW

**Comment by wjgogogo** [comment]
image

File: s2-site/docs/manual/basic/sort/advanced.zh.md [hunk 1]
```diff
@@ -3,7 +3,7 @@ title: 高级排序
 order: 1
 ---
 
-`SheetComponent` 自带高级排序组件，可根据选择组合数据提供給用户使用。
+`SheetComponent` 自带高级排序组件，可根据选择组合数据提供给用户使用。
```
Comment (wjgogogo) [inline s2-site/docs/manual/basic/sort/advanced.zh.md:6]:
```suggestion
`SheetComponent` 自带高级排序组件，可选择使用。
```

File: s2-site/docs/manual/basic/totals.zh.md [hunk 2]
```diff
@@ -5,23 +5,23 @@ order: 5
 
 ## 简介
 
-小计总计属于表的透视功能，可以給行和列分别配置小计总计。
+小计总计属于表的透视功能，可以给行和列分别配置小计总计。
```
Comment (wjgogogo) [inline s2-site/docs/manual/basic/totals.zh.md:8]:
```suggestion
小计总计属于表的透视功能，用于给行和列分别配置小计总计。
```

Comment (lcx-seima) [reply s2-site/docs/manual/basic/totals.zh.md:8]: 給 => 给

我看原文，第一句是解释，第二句是强调功能。感觉更易读。或者加一个“...用户可以给...”

**Comment by lcx-seima** [comment]
已统一图片宽度

**Comment by lijinke666** [comment]
merge 下 最新的master, 测试修了

**Review by lijinke666 — APPROVED** [review]

File: s2-site/docs/manual/introduction.zh.md [hunk 3]
```diff
@@ -214,7 +214,7 @@ cd S2
 yarn # 或者 yarn bootstrap
 
 # 调试 s2-core
-yarn core:start
+yarn core:watch
```
Comment (xingwanying) [inline s2-site/docs/manual/introduction.zh.md:217]: yarn core:start

File: s2-site/docs/manual/introduction.zh.md [hunk 4]
```diff
@@ -214,7 +214,7 @@ cd S2
...
 
 # 调试 s2-react
 yarn react:playground
```
Comment (xingwanying) [inline s2-site/docs/manual/introduction.zh.md:220]: yarn core:watch
yarn react:playground

**Pull request merged**